src/main/app/src/app/klanten/zoek/betrokkene/betrokkene-zoek.component.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * SPDX-FileCopyrightText: 2022 Atos
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+
+import {Component, EventEmitter, Input, OnInit, Output} from '@angular/core';
+import {Klant} from '../../model/klant';
+import {SelectFormFieldBuilder} from '../../../shared/material-form-builder/form-components/select/select-form-field-builder';
+import {Validators} from '@angular/forms';
+import {SelectFormField} from '../../../shared/material-form-builder/form-components/select/select-form-field';
+
+@Component({
+    selector: 'zac-betrokkene-zoek',
+    templateUrl: './betrokkene-zoek.component.html',
+    styleUrls: ['./betrokkene-zoek.component.less']
+})
+export class BetrokkeneZoekComponent implements OnInit {
```

Comment:
De naam suggereert dat je hiermee alleen een betrokkene kunt zoeken maar je kunt ook een initiator zoeken. Dus of Initiator toevoegen aan naam van component of Betrokkene wijzigen in iets anders wat zowel een betrokkene als een initiator zou kunnen zijn.

ik zou het KLantZoekComponent noemen en alleen de term betrokkene gebruiken als het ook daadwerkelijk een betrokkene is.

Reply:
Maar een initiator IS een betrokken, in open-zaak.